Разметка внутренней страницы

AlexG-23 opened the pull request:
- add markup of index (main) page
- add markup of catalog page

*need to add:
- delivery popup on the main page
- two-candles price range slider on the catalog page

---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/590651/tasks/3)

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/590651/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/590651-device-33/2/

AlexG-23 committed:
add full (almost*) markup of two Device's pages

- add markup of index (main) page
- add markup of catalog page

*need to add:
- delivery popup on the main page
- two-candles price range slider on the catalog page

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
     <title>HTML Academy: Девайс</title>
   </head>
   <body>
+    <div class="wrapper">
```
На этой странице есть секции которые тянутся на всю ширину экрана. Поэтому общую обертку-центровщик для всей страницы здесь сделать нельзя

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+      <header class="header">
 
```
Добавь центровщик внутри основных блоков, хэдера, секций, футера

LoremLipsum replied:
Хэдер надо разделить на блоки.
Здесь есть верхняя часть хэдера, див, внутри которого положи положи форму, и юзерские ссылки
И нижняя часть, видно по макету, это главная навигация, nav, внтутри которой лежит список с главными ссылками. У первой ссылки Каталог есть вложенный список, каторый показывается при наведении, смотри в стайлгайде, его тоже надо сверстать

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+      <header class="header">
 
+        <div class="header-logo">
```
Лого не надо оборачивать в див. Это ссылка внутри которой картинка и лого

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+      <header class="header">
 
+        <div class="header-logo">
+          device-logo
```
Убери текст device-logo. Для доступности можно используем альт у картинки

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+          device-logo
+        </div>
+
+        <nav class="navigation">
```
В nav оберни только главную навигацию, там где каталог, доставки и прочее

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+            <ul class="menu">
+              <li class="menu-search-field">
+                <input type="text" placeholder="Текст поиска">
```
Поиск, это форма, инпут обернутый в лейбл с текстовым описанием и кнопка для отправки формы

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+            <ul class="menu">
+              <li class="menu-search-field">
+                <input type="text" placeholder="Текст поиска">
+                <a href="#">Найти</a>
```
Это юзерские ссылки, можно обернуть просто в дивы или пока оставь так. Задай им класс

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+      </header>
+
+      <main class="main">
+        <section class="bestsellers">
```
Проще назови  slider или intro. bestsellers это как то слишком

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+        <section class="bestsellers">
+
+          <div class="bestsellers-slider">
+            <span class="bestsellers-slider-previous visually-hidden">previous</span>
```
Это кнопки следующий слайд и предыдущий, вытащи их из дива, их надо поставить абсолютом. Сверстаны должны быть все три слайда

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+          <p>Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.</p>
+
+          <a class="bestsellers-more-info overlined-link" href="#">подробнее</a>
```
Здесь основная кнопка, нужно добавить общий класс для всех основных кнопок button

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+          <a class="bestsellers-more-info overlined-link" href="#">подробнее</a>
+
+          <div class="bestsellers-slider-dots">
+            <span class="dot"></span>
```
Здесь кнопки смотри как сверстан слайдер с Фотогалерей в демке

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+            <span class="dot"></span>
+          </div>
+
+          <ul class="bestsellers-specifications">
```
Таблица, или список определений

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+        </section>
+
+        <ul class="product-categories">
```
Это следующая секция, ей нужно добавить скрытый заголовок

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+        <ul class="product-categories">
+          <li>
+            <div class="product-symbol"></div>
```
Что за дивы? Сделай ссылку внутри которой добавь картинки

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+        <ul class="product-categories">
+          <li>
+            <div class="product-symbol"></div>
+            <p class="product-name">Виртуальная реальность</p>
```
Заголовок

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+          <ul class="services-list">
+            <li>
+              <a class="services-item services-selected" href="#">доставка</a>
```
button и в остальных пункатах также

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+            <p class="product-name">Виртуальная реальность</p>
+          </li>
+          <li>
+            <div class="product-symbol"></div>
```
и в остальных пункатах также

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+            </li>
+          </ul>
+
+          <h2 class="main-title">Доставка</h2>
```
три слайда надо сверстать, пока только один

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+        </section>
+
+        <div>
```
ссылка

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+        <ul class="about">
+          <li class="about-item">
+            <h2 class="main-title">О нас</h2>
```
Почему заголовок в лишке?

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+          <li class="about-item">
+            <h2 class="main-title">О нас</h2>
+          </li>
+          <li class="about-item">Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.</li>
```
Почему текст в лишке???

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+          <li class="about-item">Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.</li>
+          <li class="about-item">Мы можем доставить ваш товар в самые отдаленные уголки России транспортными компаниями:</li>
+          <li class="about-item">
+            <ul>
```
Вот здесь только нужен список в этой секции, убери тоот список который снаружи

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+          <li class="about-item">
+            <ul>
+              <li class="about-transport-company">
+                <b>Меловые линии</b>
```
оборачивать в b не надо, это просто текст в лишке. b или спаны например он выделяют какой-то текст, а здесь весь текст выделен, просто лишняя обёртка

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+            </ul>
+          </li>
+          <li class="about-item">
+            <a class="overlined-link" href="#">подробнее о нас</a>
```
Это тоже основная кнопка, у неё должен быть общий класс button

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+          </li>
+        </ul>
+
+        <ul class="contacts">
```
Это новая секция, оборачивать заголовок и текст не нужно. Пересмотри ещё раз лекцию  и примеры в демке. И попробуй ещё раз переписать эту секцию, пока здесь править нечего, просто набор лишек и всё. Посмотри как правильно надо оформить адрес и ссылку для телефона

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+            длительными тест-драйвами, распаковками, а 
+            также акции, спецпредложения и вот это вот всё.</p>
+
+          <input class="subscribe-email-field" type="email" placeholder="Ваш e-mail">
```
У инпута должно быть текстовое описание label

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+            также акции, спецпредложения и вот это вот всё.</p>
+
+          <input class="subscribe-email-field" type="email" placeholder="Ваш e-mail">
+          <a class="subscribe-button" href="#">подписаться</a>
```
Это кнопка для отправки формы. Перепройди тренажеры по формам https://up.htmlacademy.ru/htmlcss/33/module/2/item/6

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+
+      <footer class="footer">
+
+        <div class="footer-logo">
```
лого всегда надо оборачивать в ссылку

LoremLipsum commented on index.html:
```diff
@@ -5,8 +5,260 @@
+        <div class="footer-bottom-elements">
+
+          <div class="footer-social">
+            <ul class="social">
```
Посмотри как правильно надо офрмлять социальные кнопки в демке

AlexG-23 committed:
mistakes fixing index page

- fixing of header (dividing of navigation to groups)
- add sliders of bestsellers
- add terms & definitions of products in slider
- add slider pagination buttons
- add labels to search and subscribe forms
- add address tag, where needed
- dividing by sections on full page
- fixing of footer (add sections, fix social links)

AlexG-23 committed:
mistakes fixing on both pages

- didn't add the wrappers

AlexG-23 committed:
add enter for renew files on github

learning pull requests

AlexG-23 committed:
mistakes fixing
